### PR TITLE
chore: rework dataref to css modules

### PR DESCRIPTION
**Pull request by uladzimirdev**
[epic](https://github.com/metabase/metabase/issues/50983)

### Description

PR 5/?

Rework dataref to css modules

### How to test

- create a native model
- open a sidebar on the right "learn about your data"

### CODE REVIEW

File: frontend/src/metabase/query_builder/components/SidebarContent/SidebarContent.tsx [hunk 1]
```diff
@@ -63,9 +63,14 @@ function SidebarContent({
   );
 }
 
+const PaneContent = (props: BoxProps) => {
+  return <Box pl="lg" pr="lg" {...props} />;
```
Comment (kamilmielnik) [inline frontend/src/metabase/query_builder/components/SidebarContent/SidebarContent.tsx:67]:
```suggestion
  return <Box px="lg" {...props} />;
```

File: frontend/src/metabase/query_builder/components/dataref/NodeList.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      {...rest}
+    />
+  );
+};
+
+export {
+  NodeListTitle,
+  NodeListContainer,
+  NodeListTitleText,
+  QuestionId,
```
Comment (kamilmielnik) [inline frontend/src/metabase/query_builder/components/dataref/NodeList.tsx:91]: nit: it looks like a name of a type

```suggestion
  NodeListItemId,
```

**Review by kamilmielnik — CHANGES_REQUESTED** [review]
Looks good. I found 1 small visual regression.

File: frontend/src/metabase/query_builder/components/dataref/NodeList.tsx (hunk 2 above)
Comment (uladzimirdev) [reply frontend/src/metabase/query_builder/components/dataref/NodeList.tsx]: fixed

File: frontend/src/metabase/query_builder/components/dataref/NodeList.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    </Box>
+  );
+};
+
+const QuestionId = ({ children, ...rest }: BoxProps) => {
+  return (
+    <Box
+      component="span"
+      fz="0.75em"
+      color="var(--mb-color-text-medium)"
```
[the remark this answers is not in the document]
Comment (uladzimirdev) [reply frontend/src/metabase/query_builder/components/dataref/NodeList.tsx]: image
should be fixed in [8addc69](https://github.com/metabase/metabase/pull/50911/commits/8addc693b17416fa1b96a1bfbf54ec962fbc5495)

**Review by kamilmielnik — APPROVED** [review]

**Pull request merged**